`cargo clippy -r -p script --features webgpu` fails to compile

```
$ cargo clippy -r -p script --features webgpu
    Checking script v0.0.1 (/home/oriol/src/servo/components/script)
error[E0599]: no variant or associated item named `GetWebGPUChan` found for enum `script_traits::ScriptMsg` in the current scope
   --> components/script/dom/htmlcanvaselement.rs:354:30
    |
354 |             .send(ScriptMsg::GetWebGPUChan(sender));
    |                              ^^^^^^^^^^^^^ variant or associated item not found in `ScriptMsg`

error[E0599]: no variant or associated item named `RequestAdapter` found for enum `script_traits::ScriptMsg` in the current scope
    --> components/script/dom/webgpu/gpu.rs:124:30
     |
124  |             .send(ScriptMsg::RequestAdapter(
     |                              ^^^^^^^^^^^^^^ variant or associated item not found in `ScriptMsg`
     |
     = help: items from traits can only be used if the trait is implemented and in scope
note: `dom::bindings::codegen::Bindings::WebGPUBinding::GPU_Binding::GPUMethods` defines an item `RequestAdapter`, perhaps you need to implement it
    --> /home/oriol/src/servo/target/release/build/script_bindings-998a77b5212fb470/out/Bindings/WebGPUBinding.rs:9800:1
     |
9800 | pub(crate) trait GPUMethods<D: DomTypes> {
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0599]: no variant or associated item named `SetWebGPUPort` found for enum `script_traits::ScriptThreadMessage` in the current scope
  --> components/script/messaging.rs:88:38
   |
88 |                 ScriptThreadMessage::SetWebGPUPort(..) => None,
   |                                      ^^^^^^^^^^^^^ variant or associated item not found in `ScriptThreadMessage`

error[E0599]: no variant or associated item named `SetWebGPUPort` found for enum `script_traits::ScriptThreadMessage` in the current scope
    --> components/script/script_thread.rs:1888:34
     |
1888 |             ScriptThreadMessage::SetWebGPUPort(port) => {
     |                                  ^^^^^^^^^^^^^ variant or associated item not found in `ScriptThreadMessage`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `script` (lib) due to 4 previous errors
```

--- a/components/script/Cargo.toml
+++ b/components/script/Cargo.toml
@@ -22,7 +22,7 @@ webgl_backtrace = ["canvas_traits/webgl_backtrace"]
 js_backtrace = []
 refcell_backtrace = ["accountable-refcell"]
 webxr = ["webxr-api", "script_bindings/webxr"]
-webgpu = ["script_bindings/webgpu"]
+webgpu = ["script_bindings/webgpu", "script_traits/webgpu"]
 
 [lints.rust]
 unexpected_cfgs = { level = "warn", check-cfg = ['cfg(crown)'] }

